database/00_create_table.sql

```diff
@@ -153,6 +153,29 @@ CREATE TABLE `category` (
   `name` varchar(255) NOT NULL UNIQUE
 );
 
+--
+-- Table structure for table `reviews`
+--
+
+DROP TABLE IF EXISTS `reviews`;
+CREATE TABLE `reviews` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `userId` int NOT NULL,
+  `bookInfoId` int  NOT NULL,
+  `createdAt` datetime(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6),
+  `updatedAt` datetime(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6),
+  `updatedUserId` int NOT NULL,
+  `isDelete` boolean NOT NULL,
+  `deleteUser` int DEFAULT NULL,
+  `content` varchar(255) NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `FK_529dceb01ef681127fef04d755d3` (`userId`),
+  KEY `FK_bookInfo2` (`bookInfoId`),
+  CONSTRAINT `FK_529dceb01ef681127fef04d755d3` FOREIGN KEY (`userId`) REFERENCES `user` (`id`),
+  CONSTRAINT `FK_bookInfo2` FOREIGN KEY (`bookInfoId`) REFERENCES `book_info` (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1500 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
```

Comment:
```suggestion
) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
```
특별한 이유가 없다면 1부터 시작하는게 좋을듯..?